pkg/virt-controller/watch/vm.go

```diff
@@ -612,16 +612,18 @@ func (c *VMController) startStop(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualM
 				if stateChange.Action == virtv1.StopRequest &&
 					stateChange.UID != nil &&
 					*stateChange.UID == vmi.UID {
-					log.Log.Object(vm).V(4).Info("VMI should be restarted")
 					forceRestart = true
+					log.Log.Object(vm).Infof("processing forced restart request for VMI %s with phase %s and runStrategy: %s", vmi.Name, vmi.Status.Phase, runStrategy)
```

Comment:
`vmi.Name` is redundant here. `Object(vm)` will log it.

Reply:
ohhh yeah. I was thinkign the VMI was set using generateName, but it's not. I'll fix this.